collect_app/src/main/res/layout/two_item_image.xml

```diff
@@ -27,7 +27,7 @@
             android:layout_alignTop="@+id/image"
             android:layout_toRightOf="@+id/image"
             android:textAppearance="?android:attr/textAppearanceLarge"
-            android:text="text1"/>
+            android:text="@string/text1"/>
```

Comment:
Have you ever used the ['tools' namespace](https://developer.android.com/studio/write/tool-attributes.html) before? You can use tools:text="text1 which exposes the value only to Android Studio at design time.